src/ui/server/controllers/insurance/policy-and-export/single-contract-policy/validation/rules/total-contract-value.ts

```diff
@@ -0,0 +1,70 @@
+import { FIELD_IDS } from '../../../../../../constants';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import generateValidationErrors from '../../../../../../helpers/validation';
+import { objectHasProperty } from '../../../../../../helpers/object';
+import { isNumber, numberHasDecimal } from '../../../../../../helpers/number';
+import { RequestBody } from '../../../../../../../types';
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        SINGLE: { TOTAL_CONTRACT_VALUE: FIELD_ID },
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        SINGLE: { [FIELD_ID]: ERROR_MESSAGE },
+      },
+    },
+  },
+} = ERROR_MESSAGES;
+
+const MINIMUM = 1;
```

Comment:
can we get these from constants please?

Reply:
Create a ticket here https://ukef-dtfs.atlassian.net/browse/EMS-783